CordovaLib/Classes/Private/Plugins/CDVIntentAndNavigationFilter/CDVIntentAndNavigationFilter.m

```diff
@@ -46,7 +46,7 @@ - (void)parser:(NSXMLParser*)parser didStartElement:(NSString*)elementName names
 - (void)parserDidStartDocument:(NSXMLParser*)parser
 {
     // file: url <allow-navigations> are added by default
-    self.allowNavigations = [[NSMutableArray alloc] initWithArray:@[ @"file://" ]];
+    self.allowNavigations = [[NSMutableArray alloc] initWithArray:@[ @"file://", @"app://" ]];
```

Comment:
You can change the default by using the preference `scheme` option. But if they change, the must add `allow-navigation`

Reply:
Good point, I thought to only allow the default one, but we can easily allow the configured one without an additional allow-navigation, will change it

Reply:
Since the user can enter an invalid scheme (i.e. http, https, etc) that will default to "app", instead of reading directly from settings, I've created an `appScheme` in the view controller and use that for allowing the navigation.